int8 misi
int8 wp_index
int8 action_wp